src/mongo/db/telemetry/telemetry_thread_d.cpp

```diff
@@ -72,7 +73,7 @@ Copyright (C) 2024-present Percona and/or its affiliates. All rights reserved.
 
 #define MONGO_LOGV2_DEFAULT_COMPONENT ::mongo::logv2::LogComponent::kDefault
 
-namespace mongo {
+    namespace mongo {
```

Comment:
Unnecessary leading whitespaces?

Reply:
Hm, interesting. I don't know how this happened. Maybe some glitch in clang-formatter. I'll fic this

Reply:
fixed